### PR TITLE
Adding NFT Text.

### PR DESCRIPTION
**Pull request by Carnaux**
Added the NFT text! The PR was originally in the @kalwalt ar.js v3 branch. And as I said, maybe we can make a better gif.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -127,7 +139,7 @@ You can also use GeoAR.js **without** the script, adding `gps-entity-place` enti
 <!doctype HTML>
 <html>
 <script src="https://aframe.io/releases/1.0.0/aframe.min.js"></script>
-<script src="https://raw.githack.com/jeromeetienne/AR.js/3.0.0/aframe/build/aframe-ar.js"></script>
+<script src="https://raw.githack.com/jeromeetienne/AR.js/2.1.8/aframe/build/aframe-ar.js"></script>
```
Comment (kalwalt) [inline README.md]: @Carnaux you should update the version from:
```diff
- 2.1.8
+ 3.0.0
```

Comment (kalwalt) [reply README.md]: Sorry i see now that you fixed!

**Comment by nicolocarpignoli** [comment]
I will add it on the official doc, that I'm building at: https://ar-js-org.github.io/AR.js-Docs/

thanks @Carnaux !

**Comment by Carnaux** [comment]
Ok! As soon as I get some free time, I will help you

**Pull request closed without merging**